### ISSUE TITLE
State of feature commands not properly shown

### ISSUE THREAD

**Issue by Wintermute2k6** [Icinga/icingaweb2#4071]
## Describe the bug
Feature Command is not represented correctly from the core object attributes.

## To Reproduce

Set via Director a different attribute as seen on the example Screenshot.

## Expected behavior
A proper representation is shown in Web 2.

## Screenshots
![Screen Shot 2020-02-10 at 2 36 42 PM](https://user-images.githubusercontent.com/7203835/74154488-075f5800-4c13-11ea-9a02-c0e453383b1a.png)

## Your Environment
Icingaweb 2 = 2.7.3
Icinga2 = version: 2.10.7-1
OS = Centos7
PHP 7.1.30 (cli) (built: Aug 21 2019 09:27:22) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
Director = 1.7.2

* Icinga Web 2 version and modules (System - About):
* Web browser used:
* Icinga 2 version used (`icinga2 --version`):
* PHP version used (`php --version`):
* Server operating system and version:

**Comment by widhalmt** [comment]
ref/NC/661507

**Comment by lippserd** [comment]
I think that you've enabled/disabled those feature in the past which stores the last value to `/var/lib/icinga2/modified-attributes.conf` which in turn has precedence over the configuration.

Of course this is quite crappy since it also stores the "default" value and you can't recover from that state.

**Comment by carraroj** [comment]
so i found the values in in both masters:
 ```var/lib/iciinga2/modifed-attributes.conf
var obj = get_object("Service", "t999vmsdt02!win-sawtooth")
if (obj) {
        obj.modify_attribute("enable_active_checks", true)
        obj.modify_attribute("enable_event_handler", false)
        obj.modify_attribute("enable_passive_checks", false)
        obj.version = 1581327460.158874```

How could we change this? change false -> true  in the file?

**Comment by lippserd** [comment]
Stop Icinga. Remove the host/service from the file. Start Icinga.

**Comment by lippserd** [comment]
Will be fixed with Icinga/icinga2#7986. Closing here.
